velox/type/Type.h

```diff
@@ -1300,6 +1300,18 @@ struct Array {
   Array() {}
 };
 
+template <typename ELEMENT>
+struct ArrayProxyT {
```

Comment:
It would be nice to document this struct. Is the plan to replace it with Array once the proxy is working for all types?

Reply:
yeh this will be Array eventually.
right now its there to allow using both arrayProxy and the old interface.

Reply:
It would be nice to clarify that in a comment.